README.md

```diff
@@ -40,10 +40,27 @@ pip3 install --process-dependency-links --editable .
 If you are using conda, you can create a `babyai` environment with all the dependencies by running:
 
 ```
+git clone https://github.com/mila-udem/babyai.git
+cd babyai
 conda env create -f environment.yaml
+source activate babyai
 ```
 
-Having done that, install this repository in the conda environment using the command above.
+After that, execute the following commands to setup the environment.
+
+```
+cd ..
+git clone https://github.com/maximecb/gym-minigrid.git
+cd gym-minigrid
+pip install -e .
+```
+
+The last command installs the repository as a module. Move back to the `babyai` repository and install that as a module.
```

Comment:
I am not sure that "install as a module" is something people actually say. I think this kind of installation is called "install in the development mode"

Reply:
From `pip install --help`:

` -e, --editable <path/url>   Install a project in editable mode (i.e. setuptools "develop mode") from a local project path or a VCS url.`